src/components/books/AddBook.js

```diff
@@ -1,21 +1,45 @@
-// import React from "react";
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../../redux/books/books';
 
-const AddBook = () => (
-  <article>
-    <h3>ADD NEW BOOK</h3>
-    <form>
-      <input placeholder="Book title" />
-      <select placeholder="Categorie">
-        <option value="" selected disabled hidden>
-          Categorie
-        </option>
-        <option>Action</option>
-        <option>Science Fiction</option>
-        <option>Economy</option>
-      </select>
-      <button type="submit">ADD BOOK</button>
-    </form>
-  </article>
-);
+const AddBook = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+
```

Comment:
- [ ] Please, on this part is good that you use hooks for the title, but no for the author too? On this way, you could add the author with the title when you use redux in place to make it static. 🐱